protocol/src/common/protocol.ts

```diff
@@ -3283,8 +3304,12 @@ export {
 	DidRenameFilesNotification, RenameFilesParams, FileRename, WillRenameFilesRequest,
 	DidDeleteFilesNotification, DeleteFilesParams, FileDelete, WillDeleteFilesRequest,
 	// Monikers
-	UniquenessLevel, MonikerKind, Moniker, MonikerClientCapabilities, MonikerOptions, MonikerRegistrationOptions, MonikerParams, MonikerRequest
-
+	UniquenessLevel, MonikerKind, Moniker, MonikerClientCapabilities, MonikerOptions, MonikerRegistrationOptions, MonikerParams, MonikerRequest,
+	// Type Hierarchy
+	TypeHierarchyClientCapabilities, TypeHierarchyOptions, TypeHierarchyRegistrationOptions,
```

Comment:
These shouldn't be here right now. The exports from protocol are re-exported in api so this makes them accessible as normal API now. We should only access them via Proposed (even us internally in the client for example) until they API is final (out of proposed).

Reply:
updated.